ui/app/src/modules/Content/Delete/DeleteDialog.tsx

```diff
@@ -189,6 +189,14 @@ function DeleteDialogUI(props: DeleteDialogUIProps) {
   } = props;
   const classes = deleteDialogStyles({});
   const { formatMessage } = useIntl();
+  const buttonRef = useRef<HTMLButtonElement>();
+
+  useEffect(() => {
+    if (selectedItems?.length > 0) {
+      buttonRef.current.focus();
```

Comment:
On second thought, this could be harmful for accessibility and usability. Let’s revert.